### ISSUE TITLE
Unable to create QuickSight Dataset 

### ISSUE THREAD

**Issue by seanlogan-wh** [pulumi/pulumi-aws-native#666]
### What happened?

It does not appear possible to create a [awsnative.quicksight.DataSet](https://www.pulumi.com/registry/packages/aws-native/api-docs/quicksight/dataset) resource using pulumi when specifying a physicalTableMap.  The same works using the [CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-quicksight-dataset.html) resource 

From what it appears, the interface is not defined on the pulumi resource.

### Steps to reproduce

Create a dataSet using the pulumi aws native api and run a pulumi preview.  The preview will fail.  

```
const testDataSet = new awsnative.quicksight.DataSet("testDataSet", {
    dataSetId: "data_set_id",
    name: "data_set_name",
    importMode: 'DIRECT_QUERY',
    physicalTableMap: {
        'testTable': {
            customSql: {
                dataSourceArn: dataSource.arn,
                name: "dataSourceQuery",
                sqlQuery: "select * from test_table",
                columns: [
                    {
                        name: "id",
                        type: "INTEGER",
                    },
                ],
            },
        },
    },
    logicalTableMap: {
        'testTable': {
            alias: "testTable",
            source: {
                physicalTableId: "testTable",
            },
        },
    },
});
```

Run a pulumi preview command

### Expected Behavior

The dataset will be created will be created in AWS QuickSight.

### Actual Behavior

The preview fails with the following error.  
```
error: aws-native:quicksight:DataSet resource 'testDataSet': property physicalTableMap.testTable value {<nil>} has a problem: unknown property physicalTableMap.testTable
    error: aws-native:quicksight:DataSet resource 'testDataSet': property logicalTableMap.testTable value {<nil>} has a problem: unknown property logicalTableMap.testTable
```

### Output of `pulumi about`

CLI          
Version      3.43.1
Go Version   go1.19.2
Go Compiler  gc

Plugins
NAME        VERSION
aws         5.10.0
aws         4.38.1
aws-native  0.19.0
docker      3.2.0
nodejs      unknown
random      4.8.1

Host     
OS       darwin
Version  12.6
Arch     x86_64

This project is written in nodejs: executable='/Users/<>/.nvm/versions/node/v18.7.0/bin/node' version='v18.7.0'

Current Stack: quicksight-poc

TYPE                          URN
pulumi:pulumi:Stack           urn:pulumi:quicksight-poc::quicksight-poc::pulumi:pulumi:Stack::quicksight-poc-quicksight-poc
pulumi:providers:aws          urn:pulumi:quicksight-poc::quicksight-poc::pulumi:providers:aws::default_5_10_0
pulumi:providers:pulumi       urn:pulumi:quicksight-poc::quicksight-poc::pulumi:providers:pulumi::default
aws:quicksight/group:Group    urn:pulumi:quicksight-poc::quicksight-poc::aws:quicksight/group:Group::test_group
pulumi:providers:aws-native   urn:pulumi:quicksight-poc::quicksight-poc::pulumi:providers:aws-native::default_0_19_0

Found no pending operations associated with quicksight-poc

Backend        
Name           <>
URL            s3://<>
User           <>
Organizations  

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by danielrbradley** [comment]
Hi @seanlogan-wh thanks for writing up this issue! There does seem to be some kind of an issue with how we're interpreting the `PhysicalTableMap` from [the specification](https://github.com/pulumi/pulumi-aws-native/blob/master/aws-cloudformation-schema/aws-quicksight-dataset.json#L383-L407).

This will require some investigation so will  be added to our backlog for future work.

**Comment by martinamoro** [comment]
error: aws-native:quicksight:DataSet resource 'overheadlatency': property logicalTableMap.testTable value {<nil>} has a problem: unknown property logicalTableMap.testTable
    error: aws-native:quicksight:DataSet resource 'overheadlatency': property physicalTableMap.testTable value {<nil>} has a problem: unknown property physicalTableMap.testTable
    
   Same error here, this issue still persists

**Comment by danielrbradley** [comment]
I believe this has now been fixed in the following PR:
- https://github.com/pulumi/pulumi-aws-native/pull/1363

Here's what the type looks like now:
https://github.com/pulumi/pulumi-aws-native/blob/550eb8c0c6559a06d584fc996e2ab58f7af97d69/sdk/nodejs/quicksight/dataSet.ts#L188

where the type is:
```typescript
    export interface DataSetPhysicalTableArgs {
        customSql?: pulumi.Input<inputs.quicksight.DataSetCustomSqlArgs>;
        relationalTable?: pulumi.Input<inputs.quicksight.DataSetRelationalTableArgs>;
        s3Source?: pulumi.Input<inputs.quicksight.DataSetS3SourceArgs>;
    }
```

Previously this was:
https://github.com/pulumi/pulumi-aws-native/blob/c3f606904a17fc1b484fb720ed0e306f6d0d3703/sdk/nodejs/quicksight/dataSet.ts#L188

Where the type was an empty interface:
```typescript
export interface DataSetPhysicalTableMapArgs {}
```

I'm going to close this as complete though feel free to open any follow-up issues if there's any further problems here.
